docs/marketplace/add_to_marketplace/marketplace_item_visibility_conversion.md

```diff
@@ -0,0 +1,32 @@
+---
+id: marketplace_items_visibility_conversion
```

Comment:
shall we link this page in the overview where we mention public and private mktpl?

Reply:
Done!